qutebrowser/javascript/scroll.js

```diff
@@ -22,53 +22,99 @@
 window._qutebrowser.scroll = (function() {
     const funcs = {};
 
-    funcs.to_perc = (x, y) => {
-        let x_px = window.scrollX;
-        let y_px = window.scrollY;
+    // TODO how do I clone this function in a cleaner way?
+    const call_if_frame = window._qutebrowser.webelem.call_if_frame;
```

Comment:
I want to share the `call_if_frame` function from `webelem.js` here as well, ideally without copying it. I'm not sure if this is a 'proper way' to do this, anyone have any suggestions for cleaning this up?

Reply:
Probably time to add a `javascript/utils.py` (accessible as `window._qutebrowser.utils`) and move this there?